websites/R/RF Music/presence.ts

```diff
@@ -0,0 +1,110 @@
+import { getTimestamps, timestampFromFormat } from 'premid'
+
+const presence = new Presence({
+  clientId: '1346614173517221940',
+})
+
+const cleanText = (value?: string | null) => value?.replace(/\s+/g, ' ').trim() || ''
+const defaultLanguage = 'en'
+
+async function getStrings() {
+  return presence.getStrings({ paused: 'general.paused' })
+}
+
+let cachedStrings: Awaited<ReturnType<typeof getStrings>> | null = null
+let cachedLanguage: string | null = null
+
+presence.on('UpdateData', async () => {
+  if (!document.location.pathname.includes('/')) {
+    presence.setActivity()
+    return
+  }
+
+  const language = await presence.getSetting<string>('lang').catch(() => defaultLanguage)
+
+  if (!cachedStrings || language !== cachedLanguage) {
+    cachedStrings = await getStrings()
+    cachedLanguage = language
+  }
```

Comment:
You no longer have to do caching like this, just call `getStrings` and the extension will handle the localization changes internally for you.